src/Creative/AbstractLinearCreative.php

```diff
@@ -59,7 +59,10 @@
     
     // the user activated a control to reduce the creative to its original dimensions.
     const EVENT_TYPE_COLLAPSE = 'collapse';
-    
+
+    // the user activated the close control and close the creative.
+    const EVENT_TYPE_CLOSE = 'close';
```

Comment:
There is already event type closeLinear used for linear creatives.

As i understand, this is non-standart event type, used by some ad servers? May you show me specification of event type close or add comment to your PR that this not standartised type for linear creatives, used by some ad servers?